src/py/flwr/common/fedxgb.py

```diff
@@ -0,0 +1,153 @@
+# Copyright 2020 Adap GmbH. All Rights Reserved.
```

Comment:
This file should sit in `src/py/flwr/server/strategy` rather than `common`. `common` is for functions used both by `server` and `client`.